gitbook-docs/streaming_api.md

```diff
@@ -50,3 +50,31 @@ The server MAY provide:
 `name`, `self` and `roles` MUST return the same values as provided in the `swname`, `self` and `roles` properties within the [DNS-SD advertisement](connection.md) (if implemented).
 
 `version` MUST be the same value as `version` within the associated endpoints list provided by the http `GET` request to `/signalk` within the [REST API](rest_api.md) (if implemented).
+
+## History playback
+
+The server MAY support history playback from a certain point in time with a specified rate.
+
+To create a WebSocket connection that plays back data the client uses the request parameter `startTime` to specify the start timestamp and the optional request parameter `playbackRate` to specify the rate. Rate value parameter is a floating point value with value `1` equal to real time playback and for example `0.5` to half the real time rate and `5` to five times real time rate. Omitting the `playbackRate` will result in real time playback.
```

Comment:
I still think no `playbackRate` should return the entire set of data available and then close the connection.

Reply:
I don't think WebSocket is the way to serve this use case: browser does not do anything with the output, so downloading your raw log data is not really feasible via ws. Command line tools don't work/exist and probably all other client libs need a bit of extra effort, compared to http.

Your use case makes perfect sense, just that I don't think this is way to solve it.